src/util/get-files.ts

```diff
@@ -420,6 +459,11 @@ async function explode(paths, { accepts, output }) {
     return path;
   };
 
-  const many = all => Promise.all(all.map(file => list(file)));
-  return flatten(await many(paths)).filter(v => v !== null);
+  const many = (all: string[]) => Promise.all(all.map(file => list(file)));
+  const arrayOfArrays = await many(paths);
+  return flatten(arrayOfArrays).filter(notNull);
+}
+
+function notNull<T>(value: T | null): value is T {
+  return value !== null;
 }
```

Comment:
TypeScript is a little dumb when it comes to filtering out `null` so we have to move this into a separate function. See [this answer](https://stackoverflow.com/a/46700791/266535) for more info.